backend/app.js

```diff
@@ -16,7 +16,7 @@ app.use(cors({
     credentials: true
 }));
 app.use(express.json());
-app.use(express.urlencoded({ extended: true }));
+app.use(express.urlencoded());
```

Comment:
why this change

Reply:
this is what I was saying that I commited that payload vala change but I tried to undo that commit through revert but I don't I didn't find my way but managed to undo in this commit. So it will the same the app.js (backend) won't change.

Reply:
payload vala change, i didn't get.
But if this is unexpected and want to get back, make a new commit reverting the changes

Reply:
umm what I was saying is that I made a commit changing the _app.use(express.urlencoded());_ to _app.use(express.urlencoded({ extended: true }));_ but I revert the changes while addressing this password vala issue so in this commit it will get back to normal **app.use(express.urlencoded());** which is already there in the main branch, so it won't affect.

Reply:
Oh okay, you are saying previously you were the one who made this changes is it?

Reply:
yes and now changed it back to normal

Reply:
Can you please share the PR? @rushil-b-patel 
https://github.com/krishnaacharyaa/wanderlust/pull/292/files
Here i don't see the changes

Reply:
sorry I did't get

Reply:
Just change it back to this so that I can merge
`app.use(express.urlencoded({ extended: true }));`

Reply:
were you talking about this PR ?
![image](https://github.com/krishnaacharyaa/wanderlust/assets/96254453/f72f0146-8b6d-4f83-9ff7-b8a7fba8773d)

Reply:
> umm what I was saying is that I made a commit changing the app.use(express.urlencoded()); to app.use(express.urlencoded({ extended: true }));

This never happened in the main branch, it might be in the private branch

Reply:
> Just change it back to this so that I can merge `app.use(express.urlencoded({ extended: true }));`

Just do this,

Reply:
yes it was in private repo.
You want to change from **app.use(express.urlencoded());** to this **app.use(express.urlencoded({ extended: true }));**..? @krishnaacharyaa

Reply:
any update...? @krishnaacharyaa

Reply:
I have updated long time ago with the emoji